packages/machine-emulator-tools/Dockerfile

```
@@ -0,0 +1,11 @@
+# download machine-emulator tools and create a scratch image the tar.gz contents at root
+FROM busybox as download
+ARG VERSION
```

If we don't use `bake`, I think we need a default value here too.

```suggestion
ARG VERSION=0.10.0
```

---

Isn't it safer to not have a default value? Make it required

---

I was thinking on the developer experience here.

But, at the same time, this isn't a thing the developer will interact, right?

I'll mark this comment as resolved.